### PR TITLE
GA Offline Conversion enhancements

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Made some doc enhancements

### CODE REVIEW

File: docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx [hunk 1]
```diff
@@ -59,30 +66,38 @@ rudderanalytics.track('sign up completed', {
...
-| `properties.price` | `unitPrice` | Optional |
-| `properties.userIdentifierSource` | `userIdentifierSource` | Optional |
-| `properties.conversionEnvironment` | `conversionEnvironment` | Optional |
-| `properties.gclid` | `gclid` | Optional |
-| `properties.conversionDateTime`/`originalTimestamp` | `conversionDateTime` | Required |
-| `properties.conversionValue`/`properties.total`/`properties.value`/`properties.revenue` | `conversionValue` | Optional |
-| `properties.currencyCode`/`properties.currency` | `currencyCode` | Optional |
-| `properties.orderId`/`properties.order_id` | `orderId` | Optional |
+| `properties.conversionDateTime`<br/>`originalTimestamp` <br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `conversionDateTime` | -|
+| `email` | `hashedEmail` | If **User Identifier Source** is set in the dashboard, either `email` or `phone` must be passed. |
```
Comment (utsabc) [inline docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx:72]: Would it make sense to mention full path like `traits.email` / `context.traits.email`

Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx]: done

**Review by utsabc — APPROVED** [review]

**Review by akashrpo — APPROVED** [review]

File: docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx [hunk 2]
```diff
@@ -4,7 +4,14 @@ description: Step-by-step guide on sending your event data from RudderStack to G
 aliases: ["Google Ads Offline Conversions"]
 ---
 
-[Google Ads Offline Conversions](https://support.google.com/google-ads/answer/2998031?hl=en&ref_topic=7280668) lets you measure your sale conversions in the offline mode after a user clicks your ad but does not directly proceed to sale in the online mode.
+[Google Ads Offline Conversions](https://support.google.com/google-ads/answer/2998031?hl=en&ref_topic=7280668) lets you measure your sale conversions in the offline mode after a user clicks your ad online but does not directly proceed to sale in the online mode. In other words, it helps you track the offline sales which occur when your ads lead to a click or call to your business.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx:7]:
```suggestion
[Google Ads Offline Conversions](https://support.google.com/google-ads/answer/2998031?hl=en&ref_topic=7280668) helps you track the offline sales which occur when your ads lead to a click or call to your business. It lets you measure your sale conversions in the offline mode after a user clicks your ad online but does not directly proceed to sale in the online mode. 
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/google-adwords-offline-conversions.mdx:7]: IMO this tweak conveys the idea of Offline Conversions more clearly.

**Pull request merged**